src/parser.js

```diff
@@ -73,7 +73,13 @@ const parser = yargs
   })
   .coerce('prettier', config => {
     if (typeof config === 'object') {
-      return camelcaseKeys(config)
+      const cfg = camelcaseKeys(config)
+      return Object.keys(cfg).reduce(
+        key =>
+          cfg[key] =
+            cfg[key] === 'true' || (cfg[key] === 'false' ? false : cfg[key]),
+        {},
+      )
```

Comment:
It could maybe be simpler for the sake of code lisibility to use something like https://www.npmjs.com/package/boolify ? boolify(camelcaseKeys(config)) would work as desired ! Plus it's case insensitive :)